Xamarin.Forms.ControlGallery.Android/DensityLabelRenderer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Android.Content;
+using Android.OS;
+using Xamarin.Forms;
+using Xamarin.Forms.ControlGallery.Android;
+using Xamarin.Forms.Controls;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportRenderer(typeof(DensityLabel), typeof(DensityLabelRenderer))]
+namespace Xamarin.Forms.ControlGallery.Android
+{
+	public class DensityLabelRenderer : Xamarin.Forms.Platform.Android.FastRenderers.LabelRenderer
```

Comment:
I don't think adding a renderer just for a control you're using in the control gallery is something we would like to do

Reply:
I based on ApiLabel, to get some api (native) information and help-us to create some tests...
I think this can be util for another tests...


[ApiLabel](https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.ControlGallery.Android/ApiLabelRenderer.cs)

Reply:
My bad, I thought this was in the regular Android project and not in the control gallery one :)